### DISCUSSION TITLE
Need help in setting up

### DISCUSSION THREAD

**Discussion by Ak-sky** [jbergler/hass-ttlock#141] [Q&A] [answered]
Looks like I am lost and need some guidance-

- I have Smonet smart lock Model M6-BF.
- G2 Gateway.
- I have setup home assistant and have  configured till below -
   <img width="1338" alt="image" src="https://github.com/user-attachments/assets/7efd6736-20cd-4dae-8086-9d2eee1cfed6">
- Created the account here https://open.ttlock.com/manager and registered an application which also got approved.
  <img width="577" alt="image" src="https://github.com/user-attachments/assets/0bf65fc4-59c7-47a3-984e-6a7085320546">

Now I need to know what needs to be configured and how, I did not get any webhook URL after the integration in Home Assistant.
Please help me out!

**Comment by jbergler** [comment, marked as answer]
That home assistant screenshot looks like the HACS update entity for the integration.

Now that you have the app created on the ttlock cloud, you need to actually setup ttlock in home assistant from Settings > Devices & Services > Add Integration.
